cmd/minishift/cmd/start.go

```diff
@@ -274,11 +291,25 @@ func startHost(libMachineClient *libmachine.Client) (*host.Host, string) {
 		ShellProxyEnv:    shellProxyEnv,
 	}
 
-	fmt.Printf("Starting local OpenShift cluster using '%s' hypervisor...\n", machineConfig.VMDriver)
+	fmt.Printf(" using '%s' hypervisor ...\n", machineConfig.VMDriver)
 	var hostVm *host.Host
+
+	// configuration with these settings only happen on create
+	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
+	if !isRestart {
+		fmt.Println("-- Minishift VM will be configured with ...")
+		fmt.Println("   Memory:    ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   vCPUs :    ", machineConfig.CPUs)
+		fmt.Println("   Disk size: ", units.HumanSize(float64(machineConfig.DiskSize*units.MB)))
```

Comment:
You know what would be nice to show, the OpenShift version which gets provisioned. I just had the case that I after an upgrade ended up with OpenShift 1.5.1, even though the default is 3.6.0. I had 1.5.1 in the persistent config. Knowing the OpenShift version which is getting provisioned would be nice.